### ISSUE TITLE
Add space before the colon in test results

### ISSUE THREAD

**Issue by agrabeli** [ooni/probe#1147]
According to a translator:

> A colon is hard-coded here, and for French (like many other languages) a space is missing before the colon. Punctuation marks should never be hard-coded, but appear in the source string in Transifex.

![2020-04-24 Missing space before the colon](https://user-images.githubusercontent.com/8290574/80998995-310bd200-8e44-11ea-8d0b-06b37085507b.png)

**Comment by hellais** [comment]
You mean the `:` after "Temps d'execution"?

**Comment by agrabeli** [comment]
@hellais yes, they clarified that they are referring to that after "Temps d’exécution" (Runtime).

**Comment by hellais** [comment]
This is referring to an old version of the app. Closing.
